packages/cache/src/cache.ts

```diff
@@ -91,12 +91,11 @@ export async function restoreCache(
   let compressionMethod = await utils.getCompressionMethod()
   let archivePath = ''
   try {
```

Comment:
Don't we still need to have catch for the try, so as to continue the old behavior if `getCacheEntry` throws?

Reply:
We don't need to catch again. It is caught in below catch statement. I have updated the comments for better understanding in future.